cloud-inquisitor/aws_utils.go

```diff
@@ -37,3 +43,52 @@ func AWSAssumeRole(accountID string, role string, inputSession *session.Session)
 	)
 	return assumedSession, nil
 }
+
+func DefaultLambdaMetadata(ctx context.Context) (map[string]interface{}, error) {
+	var lambdaExecutionID string
+
+	if awsContext, ok := lambdacontext.FromContext(ctx); ok {
+		lambdaExecutionID = awsContext.AwsRequestID
+	} else {
+		sessionUUID, uuidErr := uuid.NewRandom()
```

Comment:
in which case will this happen? and can we distinguish from AWS generated request id and uuid?

Reply:
@HTTP500 we could. The rational for the UUID is to be able to track all log messages for a single execution of the lambda handler. From that point of view it doesnt exactly matter IMO. But willing to add an aws/uuid designator if desired.

Reply:
I see, I am trying to suggest that we could distinguish AWS generated ID and self generated ID